hasweb/templates/index.html

```diff
@@ -45,8 +45,14 @@
     {%- for etitle, edate, eurl, edesc in [
         ('Deep Learning Conf', 'Sat, Jul 2, Bangalore', 'https://fifthelephant.talkfunnel.com/deep-learning/',
           'A mini conference on deep learning.'),
+        ('Node JS Workshop', 'Sat, Jul 9 – Sun, Jul 10, Bangalore', 'https://ramselabs.talkfunnel.com/node-js-1/',
+          "Understand NodeJS platform along with its core modules. Creating web applications using Express JS over NodeJS as web server."),
         ('The Fifth Elephant 2016', 'Thu, Jul 28 – Fri, Jul 29, Bangalore', 'https://fifthelephant.talkfunnel.com/2016/',
           "India's most renowned data science conference"),
+        ('50p 2016', 'Thu, Sep 22 – Fri, Sep 23, Bangalore', 'https://50p.talkfunnel.com/2016/',
+          "A conference about India's digital payments ecosystem"),
+        ('JuliaCon India 2016', 'Sat, Oct 22, Bangalore', 'https://juliacon.talkfunnel.com/2016/',
+          "India’s first conference on Julia language"),
```

Comment:
Change to "On the Julia language for technical computing and the surrounding ecosystem"